### ISSUE TITLE
Screen reader does not announce month and year information for the selected date button in Calendar on Windows.

### ISSUE THREAD

**Issue by msftedad** [primefaces/primereact#8518]
### Describe the bug

**Test Environment:**
OS: Windows 11 25H2 (OS Build 26200.8039)
Browser: Edge (Version 146.0.3856.84), Chrome (Version 146.0.7680.165)
Screen Readers: NVDA (2025.3.3), JAWS (2026.2603.86), Narrator
URL: https://www.primefaces.org/primereact-v8/calendar/

**User Impact:**
Screen reader users are impacted as the selected date button does not announce the month information. This makes it difficult to understand the correct context of the selected date while navigating the calendar.

**Actual result:**
Screen reader does not announce month and year information for the selected date button in Calendar on Windows.
Screen reader announces as: Table with 6 rows 8 columns rows row 5 Fr column 7 clickable 27

**Refer Attachment:** 

1. Screen reader does not announce month and year information for the selected date button in Calendar on Windows.png
2. Screen reader does not announce month and year information for the selected date button in Calendar on Windows.mp4

https://github.com/user-attachments/assets/25e122cf-505b-4fa9-9e36-e67478df951c
<img width="1920" height="1276" alt="Image" src="https://github.com/user-attachments/assets/4775332d-f201-4a47-9cf6-b826201a7c33" />

### Reproducer

https://www.primefaces.org/primereact-v8/calendar/

### System Information

```Shell
Node: 20.10.0
npm: 10.2.3
React: 18.3.1
Prime React: 10.9.7
```

### Steps to reproduce the behavior

**Pre-Requisite:** 
1. Turn on Screen Readers:
     NVDA: Ctrl + Alt + N
     JAWS: Ctrl + Alt + J 
     Narrator: Ctrl + win + Enter 
2. Verbosity: 
    NVDA: Default 
    JAWS: Beginner, Highest 
    Narrator: Default 

**Repro Steps:**

1. Open the URL: https://www.primefaces.org/primereact-v8/calendar/ in edge/chrome browser.
2. Prime react Calendar page will open.
3. Press tab key to navigate to the Inline calendar controls.
4. Press tab key to reach to the selected date control.
5. Verify whether screen reader announces month and year information for the selected date button in Calendar or not.

### Expected behavior

Screen reader should announce month and year information for the selected date button in Calendar on Windows.
Screen reader should announce: March 2026 Table with 6 rows 8 columns rows row 5 Fr column 7 clickable 27
